example/main.py

```diff
@@ -0,0 +1,50 @@
+import os
+import steps
+import time
+
+import onepasswordconnectsdk
+from onepasswordconnectsdk.models import (ItemVault, Field)
+
+op_connect_token = os.environ["OP_CONNECT_TOKEN"]
+default_vault = os.environ["OP_VAULT"]
+connect_host = os.environ["1PASSWORD_CONNECT_HOST"]
+secret_string = os.environ["SECRET_STRING"]
+
+print(steps.steps['intro'])
+
+# CREATE A NEW 1P CONNECT CLIENT
+client = onepasswordconnectsdk.client.new_client_from_environment(connect_host)
+print(steps.steps["step1"])
+
+# CREATE A NEW ITEM
+username_item = onepasswordconnectsdk.models.Item(
+    title="Secret String",
+    category="LOGIN",
+    tags=["1password-connect"],
+    fields=[Field(value=secret_string)])
+print(steps.steps["step2"])
+
+# ADD THE ITEM TO THE 1P VAULT
+posted_item = client.create_item(default_vault, username_item)
+print(steps.steps["step3"])
+
+# GIVE SOME TIME SUCH THAT THE ITEM IS FOR SURE IN THE VAULT
+time.sleep(10)
```

Comment:
This is interesting and I believe you've identified a documentation gap. What guarantees do we give when `create_item` returns?

I was personally under the impression that it would guarantee the item is created on the Connect instance, and it would not guarantee it is also created at https://1password.com yet (which it will eventually do, and most of the time very fast if the Connect server has a good internet connection and https://1password.com is up). But this code suggests otherwise.

This same documentation is required for the all API endpoints and relevant for all SDKs and integrations.

Curious where would be a good place to document this. I'd say the [Connect API reference](https://support.1password.com/connect-api-reference/), although I believe that's currently not discoverable enough (yet) from the SDKs and integrations. Shall we can link to it from the README and docstrings?

I imagine this won't be the last general documentation we'll create for the API, so I believe it'd be good to discuss a strategy more broadly. cc @nostodnayr, @jillianwilson, @verkaufer  Can I ask for your opinion on where to document this and how to make it discoverable?

Reply:
In this same general place, I imagine documenting or linking to documentation for other general behavior, such as among others trash/delete behavior for the delete endpoint, allowed item `category` values, and the meaning and usage of `type` and `purpose` keys for item fields, which also all work consistently across all integrations.

Reply:
> trash/delete behavior for the delete endpoint, allowed item `category` values, and the meaning and usage of `type` and `purpose` keys for item fields, which also all work consistently across all integrations.
Do you think this should be present, as comments, in the `.py` file, or rather in the example/general README?

Reply:
> I was personally under the impression that it would guarantee the item is created on the Connect instance.

I was under the same impression, but creating an item and fetching it immediately generates an error (you can easily test this by running the example without the 32nd line commented out). This applies to all 3 SDKs that I have written examples for.

Reply:
@hculea My suggestion would be to include those in general documentation and reference to the general documentation from the integrations and SDKs. The general documentation could be the Connect API reference.

For two reasons:
1. Many users will use more than 1 integration or SDK. Having a separate documentation for those things that hold for all integrations makes it easier for them to understand this holds for all their use cases.
2. Having one place makes it way easier for us to maintain the documentation. Changes to Connect only need updating in Connect.

There's one disadvantage, which is that users would be looking at two different documentations (Connect API docs and the documentation for the integration/SDK) in order to use the SDK/integration. I believe this won't be problematic _if_ we link to the general documentation from the integration documentation so that it is easily discoverable from the various integrations.

Reply:
Sounds like a plan! I'll look into including this in the Connect API reference. I'll start working on a draft tomorrow morning.